runfullday.sh

```diff
@@ -95,7 +95,7 @@ case $NIC_SPD in
 esac
 
 unset STREAM_LIST
-DEFAULT_STREAM_LIST="10, 1000, 10000, 100000, 1000000"
+DEFAULT_STREAM_LIST="10, 1000, 10000, 100000, 1000000? "
```

Comment:
Guess this was added by accident

Reply:
No this was added since this didn't match with other similar queries. Its there to match the format of the questions that script asks.

Reply:
Guess also the spaces in the default list should be removed, else it will also fail execution:

This should work:

unset STREAM_LIST                                                                                                                                                       
DEFAULT_STREAM_LIST="10,1000,10000,100000,1000000"                                                                                                                      
echo -n "Enter L2/L3 streams list, \"$DEFAULT_STREAM_LIST\"(default)? "                                                                                                 
read STREAM_LIST                                                                                                                                                        
STREAM_LIST=${STREAM_LIST:-$DEFAULT_STREAM_LIST}

Reply:
I see it now, sorry that was a last minute change i thought would work, ofcourse I should have been more careful, sorry for wasting your time.